script/Ciel/Day 7/07_01_CIEL07_5.txt

```diff
@@ -797,12 +809,12 @@ I do appreciate what Arcueid is doing.
 [sha:3f8cbc7ba24e004f005dd348fb0687e3efa530b4]{
 -- Page 489, Offset 15974.
 -- 街に残っている吸血鬼だって野放しにできない。
-The vampires who still remain in the city can't just be left alone.
+The vampires still in this city can't just be left to their own devices.
 }
 [sha:e740fc3a8a34463658193fe04d1bf750227e1068]{
 -- Page 490, Offset 15975.
 -- けど、だからといってコイツに手を貸すのは別問題だ。
-But still, that is a completely separate matter from me teaming up with her.
+Still, that doesn't mean I can just agree to help her on the spot.
```

Comment:
'on the spot' moves the problem from what it actually is. That makes it sound like his problem is that the current time and place is the problem, not the actual act of teaming up with her.